regulations/static/regulations/css/scss/partials/_footer.scss

```diff
@@ -70,29 +76,43 @@ footer {
         }
     }
 
-    .footer-middle {
+    .footer-middle {    
+
         & > * {
             padding: 0;
 
-            @include custom-max($mobile-max / 1px) {
+            @include custom-max($tablet-max / 1px) {
                 padding: 0 $spacer-4;
             }
 
             @include screen-lg {
-                max-width: 90%;
+                max-width: calc(90% - #{$spacer-3});
+                padding-left: $spacer-3;
             }
         }
 
-        @include custom-max($mobile-max / 1px) {
+        @include custom-max($tablet-max / 1px) {
             padding: $spacer-4 0;
         }
     }
 
     .footer-right {
+
+        @include custom-max($tablet-max / 1px) {
+            flex: none;
+        }
+
```

Comment:
the negative margin is pulling elements outside of their parent: 
![Screen Shot 2021-09-01 at 11 10 09](https://user-images.githubusercontent.com/1651907/131700328-6e97a980-0453-4e5a-af00-40719bce7091.png)

I think we can get the same exact effect and keep everything inside of their containing elements by removing the negative margin left for all children and adding this to `.footer-right`:

```suggestion
    padding: 0 16px;
    align-items: start;
    justify-content: start;
```

Reply:
@PhilR8 I know it pulls outside of the parent, but the issue is really that the middle element is too wide. The footer columns need to align with the content columns above them so they don't look off. Pulling the right side outside of its parent seemed less awkward than adjusting the middle column when I played with it, but open to suggestions.

Reply:
Adding the padding to footer-right will make the middle narrower.  Right now the footer is set up with flexbox so that `.footer-left` and `.footer-right` have a defined flex-basis value (which is like a min-width).  `.footer-middle` doesn't - it will expand or contract to fill the remaining width.  Adding padding to `.footer-right` will make the middle narrower, while also having the right side of the footer match the width of the right sidebar in `.homepage-main-content`.

If you remove the negative margin and add the padding, everything looks exactly the same, with the benefit of having everything stay within their containing elements.

Reply:
I've made the tweaks - let's see if this .mov file below works:

https://user-images.githubusercontent.com/1651907/132902810-0375fd6e-9d98-4892-b053-5f07a6c4830b.mov

The footer should work basically the same as you had it, but `footer-right` keeps everything in its container, the middle element is narrower, and everything lines up nicely with the right sidebar as well.  

I've also done some tweaks to the external link icon - I had added padding for its use elsewhere but it introduced a side effect here, so I've just removed the padding.